homeassistant/components/huawei_lte/config_flow.py

```diff
@@ -276,11 +276,12 @@ async def async_step_ssdp(
         if TYPE_CHECKING:
             assert discovery_info.ssdp_location
         url = url_normalize(
-            discovery_info.upnp.get(
-                ATTR_UPNP_PRESENTATION_URL,
-                f"http://{urlparse(discovery_info.ssdp_location).hostname}/",
-            )
+            discovery_info.upnp.get(ATTR_UPNP_PRESENTATION_URL)
+            or f"http://{urlparse(discovery_info.ssdp_location).hostname}/"
```

Comment:
why change this to `or` instead of using `get`'s default argument?

Reply:
Because the `.get` might give us an empty string, and a fallback is desirable for that, too.